runtime/lib/component/flora.js

```diff
@@ -41,6 +42,21 @@ Flora.prototype.handlers = {
   }
 }
 
+Flora.prototype.remoteMethods = {
+  'yoda.debug.heap_snapshot': function (reqMsg, res) {
+    logger.info('take heapsnapshot', reqMsg)
+    var storePath = reqMsg[0]
+    if (!path.isAbsolute(storePath)) {
```

Comment:
* It seem `takeSnapshot` will log data in a default path named `${process.cwd()}/Profile-${Date.now()}`, thus whether this function also can support none path input? For some invoker, they may dont know whats path should be put in. (Wrapper can set another default path which is different from JS-Engine.)
* In some case, I may prefer to set a fixed filename of output. So, could this requirement be fullfiled?

Reply:
I'd suggest a simple pattern to generate snapshots like coredumps.